pkg/cmd/pipelinerun/list_test.go

```diff
@@ -22,18 +22,76 @@ import (
 	"github.com/jonboulle/clockwork"
 	"github.com/knative/pkg/apis"
 	"github.com/spf13/cobra"
-	"github.com/tektoncd/cli/pkg/testutil"
+	tu "github.com/tektoncd/cli/pkg/testutil"
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
 	"github.com/tektoncd/pipeline/pkg/reconciler/v1alpha1/pipelinerun/resources"
 	"github.com/tektoncd/pipeline/test"
 	tb "github.com/tektoncd/pipeline/test/builder"
 	corev1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 func TestListPipelineRuns(t *testing.T) {
-	now := time.Now()
-	prs := pipelineRuns(now)
+	clock := clockwork.NewFakeClock()
+
+	var (
+		ns          = "namespace"
+		runDuration = 1 * time.Minute
+
+		pr1name         = "pr1-1"
```

Comment:
nit: Those could even be inlined

Reply:
`go fmt` formatting it like that only :joy_cat:

Reply:
By inline I meant no var and using it in the function call directly, it would reduce the noise 😊

Reply:
We could :thinking: , however, it makes the test input more readable, inst it? 
I mean all test data appears first rather than in the `tb` noise? :sweat_smile:

Reply:
fixed the vars section @vdemeester . however, the test needs 3 vars so I kept it there :sweat_smile: